Source/App/EbAppMain.c

```diff
@@ -209,14 +299,14 @@ int32_t main(int32_t argc, char* argv[])
                             exitConditionsOutput[instanceCount] = ProcessOutputStreamBuffer(
                                 configs[instanceCount],
                                 appCallbacks[instanceCount],
-                                (exitConditionsInput[instanceCount] == APP_ExitConditionNone) || (exitConditionsRecon[instanceCount] == APP_ExitConditionNone) ? 0 : 1);
-                        if (((exitConditionsRecon[instanceCount] == APP_ExitConditionFinished || !configs[instanceCount]->reconFile) && exitConditionsOutput[instanceCount] == APP_ExitConditionFinished && exitConditionsInput[instanceCount] == APP_ExitConditionFinished) ||
-                            ((exitConditionsRecon[instanceCount] == APP_ExitConditionError && configs[instanceCount]->reconFile) || exitConditionsOutput[instanceCount] == APP_ExitConditionError || exitConditionsInput[instanceCount] == APP_ExitConditionError)) {
+                                inputDone || (exitConditionsRecon[instanceCount] == APP_ExitConditionNone) ? 0 : 1);
```

Comment:
It should be "!inputDone || ......", because inputDone indicates whether EOS has been triggerred.

Reply:
agree.